config/config.go

```diff
@@ -928,6 +928,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.adman.endpoint", "http://pub.admanmedia.com/?c=o&m=ortb")
 	v.SetDefault("adapters.admixer.endpoint", "http://inv-nets.admixer.net/pbs.aspx")
 	v.SetDefault("adapters.adnuntius.endpoint", "https://ads.adnuntius.delivery/i")
+	v.SetDefault("adapters.adnuntius.extra_info", "https://europe.delivery.adnuntius.com/i")
```

Comment:
This is the source of the merge conflict. If you merge in the master branch, you'll see that we recently moved adapter config to the yaml files.

Reply:
Yes, I think it's merged incorrectly. There are should not be any changes in config.go. There are no bidder configs anymore.